tools/abel/model/model_builder.py

```diff
@@ -123,9 +123,10 @@ def FromBuildingConfig(cls, site: Site,
       model_builder.AddReportingEntitiesFromEntityInstance(entity_instance)
     return model_builder
 
+  # pylint: disable=line-too-long
   # TODO(b/234630862) Refactor Build method for readability.
   def Build(self) -> None:
-    """Connects all entities to a site, fields to entities, and entities to entities based on attributes."""
+    """Connects ABEL graph with Guids as edges."""
```

Comment:
I prefer the prefer the previous docstring here

Reply:
Done. Same as above.